docs/index.html

```diff
@@ -179,8 +179,8 @@ <h1 id="api-instantiation">Instantiation</h1>
 
 <p>Ever need to loop through some dates to find the earliest or latest date?  Didn't know what to set your initial maximum/minimum values to? There are now two helpers for this to make your decision simple:</p>
 
-<p><pre><code class="php">echo Carbon::maxValue();                               // '2038-01-18 22:14:07'
-echo Carbon::minValue();                               // '1901-12-13 15:45:52'
+<p><pre><code class="php">echo Carbon::maxValue();                               // '9999-12-31 23:59:59'
```

Comment:
That really depends on system OS (32 or 64bit).
https://en.wikipedia.org/wiki/Year_2038_problem
Should we also comment that?

Reply:
I was exactly thinking of that. :)